### PR TITLE
Changes required to run on EMR 4.7.2

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

**Comment by dcadams** [comment]
@stvstnfrd

File: batch/provision.yml [hunk 1]
```diff
@@ -5,14 +5,13 @@
   gather_facts: False
   vars:
     # The following vars should probably be overridden
-    - name: ETL
+    - name: Lagunita 4.7.2_6 EMR Cluster
```
Comment (stvstnfrd) [inline batch/provision.yml:8]: I'm not a huge fan of the `_6` bit..
That's just the 6th iteration of this?
It's a bit confusing, given that the other numbers refer to an actual version number, but the suffix is arbitrary.

**Comment by stvstnfrd** [comment]
Any idea why we had to make so many changes on top of what edX had?

File: batch/provision.yml (hunk 1 above)
Comment (dcadams) [reply batch/provision.yml:8]: Yes, there doesn't seem to be a way of deleting terminated clusters otherwise I wouldn't have used this.

**Comment by dcadams** [comment]
Some of it is iterating until a configuration that works for us. Some of it is lack different versions and some of it is lack of a fully functional config from upstream.

In any event, this is the config that is currently used for Lagunita analytics.

**Pull request merged**